docs/source/development/install/install.rst

```diff
@@ -65,10 +65,8 @@ The next step is it create this environment, called *landlab_dev*.
 .. code-block:: bash
 
    $ conda create -n landlab_dev python
-   $ conda install -n landlab_dev --file=requirements.txt -c conda-forge
-   $ conda install -n landlab_dev --file=requirements-testing.txt -c conda-forge
-   $ conda install -n landlab_dev --file=requirements-notebooks.txt -c conda-forge
-   $ conda install -n landlab_dev --file=requirements-dev.txt -c conda-forge
+   $ conda create -n landlab_dev -c conda-forge --file=requirements.txt --file=requirements-dev.txt --file=requirements-testing.txt --file=requirements-notebooks.txt
```

Comment:
If you remove the "=" after each `--file` flag, then shell completion works.

Reply:
@mdpiper should I let only the first --file this way?
```
conda create -n landlab_dev -c conda-forge --file requirements.txt requirements-dev.txt requirements-testing.txt requirements-notebooks.txt
```

Reply:
No, unfortunately each requirements file requires its own `--file` flag. (I've tried this before and it failed.)

Reply:
Alright @mdpiper I commit the changes :smiley:

Reply:
Thanks @sebastien-lenard!